test/__init__.py

```diff
@@ -1,3 +1,6 @@
+import os
+
 from dotenv import load_dotenv
 
-load_dotenv("../secrets/test.env")
\ No newline at end of file
+test_directory_path = os.path.dirname(os.path.abspath(__file__))
+load_dotenv(f"{test_directory_path}/../secrets/test.env")
```

Comment:
@josiahdc I do see that you exclude `.env` files and secrets directories in the `.gitignore` file. However, my understanding of best practice is to only include safe template files in the project dir and load all "real" `.env` and secrets files from somewhere outside the project dir (e.g., from `~/.mycelium`). Otherwise, one slip-up at any point in the `.gitignore` and secrets get exposed without people even realizing it. (Also, if you want to do something innocuous like transfer a copy of a working directory to somebody, you have to manually go through and find+scrub the secrets, otherwise, again, you expose secrets without realizing it.)

Reply:
I definitely take you point @chrislrobert - I wonder if there's a way to make this change without cluttering the user's home directory. We're also going to have to make this change for all other repos involved in the project, since they use a similar system - I just created a ticket for this so we can figure out a good solution and implement it across the whole project to keep things consistent.